_pages/en_US/alternate-exploits.txt

```diff
@@ -59,8 +61,13 @@ Continue to [Installing boot9strap (kartdlphax)](installing-boot9strap-(kartdlph
 
 This method requires a compatible DS flashcart to be flashed with ntrboot, which works on all 3DS-family consoles regardless of model or region.
 
-If none of the above options are available to you, this is the cheapest available option. If you don't already have a compatible cartridge, a preflashed ntrboot-compatible cartridge can be purchased [here](https://www.nds-card.com/ProShow.asp?ProID=575) for roughly $20 USD.
+If none of the above options are available to you, this is the cheapest available option. If you don't already have a compatible cartridge, a pre-flashed ntrboot-compatible cartridge can be purchased [here](https://www.nds-card.com/ProShow.asp?ProID=575) for roughly $20 USD.
 
 Continue to [ntrboot](ntrboot)
 {: .notice--primary}
 
+#### Hardmod
+
+This method requires a Hardmod and a computer
```

Comment:
sentence needs a period at the end
hardmod shouldn't be capitalized
it would be good to explain what a hardmod is